spec/requests/api/orders/update_spec.rb

```diff
@@ -0,0 +1,44 @@
+RSpec.describe 'Update an order with a PUT /api/orders/:id', type: :request do
+
+  
+  let(:user) { create(:user) }
+  let(:product_1) { create(:product, name: 'Tshirt') }
+  let(:product_2) { create(:product, name: 'Football') }
+  let(:product_3) { create(:product, name: 'Scarf') }
+  let(:order) { create(:order) }
+  let!(:order_item_1) { create(:order_item, order:, product: product_2) }
+  let!(:order_item_2) { create(:order_item, order:, product: product_3) }
```

Comment:
Why is there a `:` in the second argument of `create`?

Reply:
I understand this creates the order item  for product 2 product 3. @maxarvid can you advise help / advise how to re-write. When I try remove tests all fail

Reply:
So this is not a problem, I was looking for an explanation of why we can set a hash value to nothing. I've made a post in Slack explaining why we can omit the value here.